pkg/buildinfo/buildinfo.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package buildinfo provides high-level build information injected during
+// build.
+package buildinfo
+
+var (
+	// BuildID is the unique build identifier.
+	BuildID string = "unknown"
+
+	// BuildTag is the git tag from which this build was created.
+	BuildTag string = "unknown"
```

Comment:
is it possible to also get the commit sha for when a build is done from head

Reply:
Does `go tool buildid` work for your use case???

Reply:
`git describe --dirty` (which I'm using) does what I think you want, some examples:

```text
v0.1.1
v0.1.1-ab28ced
v0.1.1-ab28ced-dirty
```

Reply:
@jeremyfaller we explicitly want the _Cloud Build_ Build ID here (it's a UUID).